### ISSUE TITLE
draw vs drawTwo vs drawFromDiscard vs drawFromPlayer vs drawExclamation

### ISSUE THREAD

**Issue by Katacka** [vargaspu21/Bang_GameState_Final#11]
It seems like you might have an excess number of draw methods. Specifically, draw vs drawTwo sounds like something which could be handled based on an integer parameter specifying the number of cards which should be drawn. If special behavior like vulture sam only occurs in a drawTwo scenario, I would recommend separating that from the drawTwo method as such implementation could grow convoluted.

**Comment by hujo22** [comment]
That is true. In fact, drawTwo for the most part applies to the drawing at the start of each player's turn. We will decide to change the name to something that implies that as well as add a method to draw a specific number of cards!

We may have explained the effects of Vulture Sam incorrectly. His effect occurs whenever another player dies (so this mainly happens during the card-playing phase). He takes all of the cards of that player.
